src/actions/arcActions.ts

```diff
@@ -163,24 +159,17 @@ export function redeemReputationFromToken(scheme: Scheme, addressToRedeem: strin
       };
       const gasEstimate = await arc.web3.eth.estimateGas(txToSign);
       txToSign.gas = gasEstimate;
-      console.log(`estimated gas cost           : ${gasEstimate  * gasPrice}`);
-      // console.log(txToSign);
       // if the gas cost is higher then the users balance, we lower it to fit
       const userBalance = await arc.web3.eth.getBalance(redeemerAddress);
-      console.log(`balance of ${redeemerAddress}: ${userBalance}`);
       if (userBalance < gasEstimate * gasPrice) {
         txToSign.gasPrice = Math.floor(userBalance/gasEstimate);
       }
-      console.log(`estimated gas cost after adjusting: ${gasEstimate  * txToSign.gasPrice}`);
-      const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
-      // console.log(signedTransaction);
+      // const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
       dispatch(showNotification(NotificationStatus.Success, "Sending redeem transaction, please wait for it to be mined"));
-      const txHash = await arc.web3.utils.sha3(signedTransaction.rawTransaction);
-      console.log(`transaction hash: ${txHash}`);
+      // const txHash = await arc.web3.utils.sha3(signedTransaction.rawTransaction);
       try {
-        const receipt  = await arc.web3.eth.sendSignedTransaction(signedTransaction.rawTransaction);
+        // const receipt  = await arc.web3.eth.sendSignedTransaction(signedTransaction.rawTransaction);
```

Comment:
This was the crucial line of this action, to send the transaction.

if the `receipt` const is not being used, plz change to

`await arc.web3.eth.sendSignedTransaction(signedTransaction.rawTransaction);`

and not remove the whole line :-)

Reply:
done